common/src/main/resources/xsl/thunibib-mods-filter-supported.xsl

```diff
@@ -143,4 +142,28 @@
   <xsl:call-template name="copy-and-apply" />
 </xsl:template>
 
+<xsl:template match="//mods:subject">
```

Comment:
this template should be content of
<xsl:template match="mods:subject[mods:topic]|mods:subject/mods:topic[1]">
... in line 133 or adopted to <xsl:template match="mods:subject" >, so that we have one template for subjects